lib/message.rb

```diff
@@ -0,0 +1,45 @@
+require 'pg'
+require_relative 'db'
+
+class Message
+
+  attr_reader :id, :message, :date
+
+  def initialize(id:, message:, date:)
+    @id = id
+    @message = message
+    @date = date
+  end
+
+  def self.all
+    connection = Db.connect
+    result = connection.exec("SELECT * FROM peeps")
+    result.map do |message|
+      Message.new(id: message['id'], message: message['message'], date: message['date'])
```

Comment:
This business of wrapping return values in instances of Message requires a bit of mental gymnastics. Well done for doing that.